it.sh

```diff
@@ -23,7 +23,7 @@
 set -e
 
 # Enable for debugging
-#set -x
+set -x
```

Comment:
Have enabled logging here to debug the other failing integration-test. Will revert this change before merging this PR.

Reply:
Have reverted that change.